### PR TITLE
Maintainer docs: Managing PRs

### PR DESCRIPTION
**Pull request by e0d**
PR onto the branch to capture some agree to standards for managing PRs.

### CODE REVIEW

File: source/developers/how-tos/ongoing-maintainers-tasks.rst [hunk 1]
```diff
@@ -35,15 +35,27 @@ GitHub Notifications
...
+
+One of the scarest resources of an open-source project is contributor and maitainer time.  It is ideal that once a review starts, that it can be finalized as a single piece of work, say within the boardaries of a single sprint.
+
+As a maintainer, your goal should be keeping the time to review and finalize a PR predictable and as short as reasonable.  Finalizing a PR can mean rejecting it. Knowing that a PR won't be accepted and why provides the contributor appropriate feedback and let's them decide what to do next.
+
+If your team schedules reviews for future sprints, providing that information to the author is helpful to set their expectations and plan accordingly. Leave a comment indicating when you expect the review to occur and ensure that the status on the contributions board is "Scheduled for Eng Review."
+
+When reviewing a PR, we want to focus on reducing the back-and-forth that increases lead time.  In our globally distributed project, a comment and response cycle can easily take 24 hours.  Ideally, both contributor and reviewer can both focus their attention on finalizing a PR at the same time.
```
Comment (sarina) [inline source/developers/how-tos/ongoing-maintainers-tasks.rst:54]: > Ideally, both contributor and reviewer can both focus their attention on finalizing a PR at the same time.

I'm not sure the purpose of this sentence. What action or knowledge am I supposed to take from it? Should reviews _only_ occur if this is the case? What if it's not? I think it's obvious that this is the ideal case but I'm missing the goal of including this statement here.

File: source/developers/how-tos/ongoing-maintainers-tasks.rst [hunk 2]
```diff
@@ -35,15 +35,27 @@ GitHub Notifications
...
+
+It is recommended that reviewers approve the GitHub actions if approval is required.  Github requires approval from new contributors to any repository, so even folks who have contributed to the project, but not the particular repository, require approval.
+
+It is recommended that when GitHub detects that the PR branch is out of date with the base, that the reviewer update the branch using a merge commit.  This keeps the history clear, but reduces days of lead time over pushing this back to the original author.  However, reviewers should ask authors to resolve any conflicts between their branch and the base.
```
Comment (sarina) [inline source/developers/how-tos/ongoing-maintainers-tasks.rst:58]:
```suggestion
When GitHub detects that the PR branch is out of date with the base, it is recommended that the reviewer update the branch using a merge commit.  This keeps the history clear and reduces days of lead time over pushing this back to the original author.  However, reviewers may ask authors to resolve any conflicts between their branch and the base.
```

Some suggested changes here. I think if a maintainer wants to resolve conflicts, they should be able to do that.

**Review by sarina — COMMENTED** [review]
@e0d overall this looks good. The recommendations are along the lines of the ones I believe we've agreed on for PR review. I have a few questions, then I will approve.

**Comment by sarina** [comment]
@e0d pinging on this

File: source/developers/how-tos/ongoing-maintainers-tasks.rst (hunk 1 above)
Comment (e0d) [reply source/developers/how-tos/ongoing-maintainers-tasks.rst:54]: If a reviews two weeks after the PR comes in, b has moved on, b finishes their sprint and returns to the PR.  A is now in their sprint and doesn't get back to it for another week, and so on.  Ideally, everyone can focus on completing the review in the same timeslice and avoid delays.

Comment (sarina) [reply source/developers/how-tos/ongoing-maintainers-tasks.rst:54]: OK, I'm not entirely sure I understand my original nitpick here anyway. Makes sense.

**Pull request merged**